doc/bt/ZC_UPDATED_PCAPPEARANCE.bt

```diff
@@ -17,4 +18,4 @@ short op;
 int sequence;
 
 int handle;
-COMMANDER commander;
+APPEARANCE_PC appearancePc;
```

Comment:
I think this makes a lot of sense now.

Reply:
Yea, it does. I'm all for this change if that's also how the client handles it, even if restructuring like this is naturally sub-optimal, since the same name suddenly means something different, and a whole lot of code changes without any *actual* change, which can easily lead to mistakes and bugs.